specification.md

```diff
@@ -444,9 +444,12 @@ deliver to the Agent (such as for example a new remote configuration).
 
 The default polling interval when the Agent does not have anything to deliver is 30
 seconds. This polling interval SHOULD be configurable on the Client.
+If the server sets OpAMPConnectionSettings.heartbeat_interval_seconds, the client MUST
```

Comment:
This section applies to http transport only. I think we need to add something in the websocket section too.